src/basics/variable/d.ladziata/homeTask2_Logical_operators/logical_operators

```diff
@@ -0,0 +1,78 @@
+/* 
+Logical OR: Returns expr1 if it can be converted to true; otherwise, returns expr2
+Logical AND: Returns expr1 if it can be converted to false; otherwise, returns expr2
+*/
+
+/*                                                        Task 1                                                 */
+//console.log(NaN || 2 || undefined); //answer : 2
+//console.log(NaN && 2 && undefined); //answer : NaN
+//console.log(1 && 2 && 3); // answer: 3
+//console.log(!1 && 2 || !3); // answer: false
+//console.log(25 || null && !3); //answer: 25
+//console.log(NaN || null || !3 || undefined || 5); //answer: 5
+//console.log(NaN || null && !3 && undefined || 5); //answer: 5
+//console.log(5 === 5 && 3 > 1 || 5); //answer: true
+
+/*                                                        Task 2                                                 */
+/* const hamburger = 3;
+const fries = 3;
+const cola = 0;
+const nuggets = 2;
+
+if (hamburger === 3 && cola || fries === 3 && nuggets) {
```

Comment:
add some explanation 
1. hamburger === 3 - true,
 2. true && cola - 0, 
 3. fries === 3 - true, 
 4. true && nuggets - 2,  
 5. 0 || true - true. Function condition is true that why 'Done!' is displayed.